lib/statusbar-summary-view.coffee

```diff
@@ -27,16 +27,24 @@ class StatusBarSummaryView
     @tile = statusBar.addRightTile({item: el, priority: 100})
 
 class StatusBarSummary extends View
+  initialize: ->
+    @on 'click', '.linter-summary-container', ->
+      console.log("moveToNextMessage")
```

Comment:
My first thought was to make a new virtual class/view and inherit the different linter view (statusbar, inline...) from it but it's not Java ^^ and if you look at the different linter view, there are not so much duplicate/shared code between them (except the moveToNextMessage function and some little things).

And just to be sure, about https://github.com/AtomLinter/Linter/issues/424, when you are saying: `Next Message command moves the cursor in every editor, not just active editor`, it means that we should be able to go through different files not just the active one, right?